### PR TITLE
test/unignore tests

### PR DESCRIPTION
**Pull request by artHalMirantis**
verification of the followed tests was done
tests can be included to the run
part of the tests were change with appropriate commit or/and new issue assigned to it

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy [hunk 1]
```diff
@@ -892,7 +892,7 @@ class ProtectedPathSpec extends HealthCheckSpecification {
                 it.state == IslChangeType.FAILED
             }.size() == broughtDownPorts.size() * 2
         }
-
+        print(flow)
```
Comment (niksv) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy:895]: Please delete

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy [hunk 2]
```diff
@@ -266,7 +266,8 @@ class IslReplugSpec extends HealthCheckSpecification {
         database.resetCosts(topology.isls)
     }
 
-    @Ignore("https://github.com/telstra/open-kilda/issues/3780")
+//    @Ignore("https://github.com/telstra/open-kilda/issues/3780")
+//    test need supervision (on hardware environment) cause vm is not match to the test requirements
```
Comment (niksv) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy:270]: why comment? not delete?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy [hunk 3]
```diff
@@ -493,7 +493,7 @@ mode with existing flows and hold flows of different table-mode types"() {
     }
 
     @Tags([LOW_PRIORITY])
-    @Ignore("https://github.com/telstra/open-kilda/issues/3961")
+//    @Ignore("https://github.com/telstra/open-kilda/issues/3961")
```
Comment (niksv) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy:496]: same

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy [hunk 4]
```diff
@@ -683,7 +683,8 @@ mode with existing flows and hold flows of different table-mode types"() {
     }
 
     @Tags([LOW_PRIORITY])
-    @Ignore("https://github.com/telstra/open-kilda/issues/4043")
+//    @Ignore("https://github.com/telstra/open-kilda/issues/4043")
+//    unignored. Test needs supervision next build. On phys environment.
```
Comment (niksv) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy:687]: same

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy [hunk 5]
```diff
@@ -71,6 +71,7 @@ class LagPortSpec extends HealthCheckSpecification {
             logicalPortNumber > 0
             portNumbers.sort() == portsArrayCreate.sort()
             it.lacpReply == lacpReply
+            print(it.lacpReply)
```
Comment (niksv) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy:74]: print in tests is a bad practice

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy (hunk 1 above)
Comment (artHalMirantis) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy]: resolved with the latest commit

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy (hunk 5 above)
Comment (artHalMirantis) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy]: resolved

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy (hunk 4 above)
Comment (artHalMirantis) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy]: resolved

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy (hunk 3 above)
Comment (artHalMirantis) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy:496]: accidentally unignore, test should be ignored, and removed after single table support ends

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy (hunk 2 above)
Comment (artHalMirantis) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy]: left it due to the test is not tested on hardware env, so to take supervision after
but you are right, delete it

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy (hunk 3 above)
Comment (niksv) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy:496]: It is still unignored

**Review by niksv — APPROVED** [review]

**Review by dmitrii-beliakov — APPROVED** [review]

**Pull request merged**